.travis.yml

```diff
@@ -46,7 +47,7 @@ before_install:
   galaxy_kickstart`
 - docker ps
 install:
-- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend[testing]'
+  - sudo su $GALAXY_TRAVIS_USER -c 'pip install --user https://github.com/mvdbeek/bioblend/archive/test_data.zip pytest'
```

Comment:
what about
`- sudo su $GALAXY_TRAVIS_USER -c 'cd && git clone https://github.com/mvdbeek/bioblend.git --branch test_data && pip install bioblend/ --user ` 
and we revert to
`- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend` when your PR is merged ?